docs/community/content/contribute/committer.en.md

```diff
@@ -25,25 +25,19 @@ After a contributor participate ShardingSphere community actively, PMC and Commi
 
 1. Weekly rotating committers needs to check the list of Pull Requests and issues to be processed in the community on a daily basis.
 
- - Including(mark issue lable, reply issue, close issue).
- - Assign issue to the appropriate committer, namely assignee.
+ - Including label issue, reply issue, close issue;
+ - Assign issue to the appropriate committer, namely assignee;
  - The rotation period is one week. 
  
->Committer rota will update in the group chat, this position is aims to bring out the sense of community ownership and greater the sense of belonging, the Apache ShardingSphere is belongs to every PMC/Committer!
-
 2. After a committer is assigned with an issue, the following work is required:
 
- - Estimate whether it is a long-term issue. If it is, please mark it as pending.
- - Add issue labels, such as bug, enhancement, discussion, etc.
- - Add Milestone labels.
+ - Estimate whether it is a long-term issue. If it is, please mark it as pending;
+ - Add issue labels, such as bug, enhancement, discussion, etc;
+ - Add Milestone.
  
 3. Pull Request that committer submits needs to mark labels and milestone based on the type and time.
```

Comment:
Pull Request that committer submits needs to mark labels and milestone based on the type and `release period`